### PR TITLE
[PURCHASE-1238, PURCHASE-1248] PartnerCard and SellerInfo adjustments

### PR DESCRIPTION
**Pull request by lilyfromseattle**
# [PURCHASE-1238] Label above the partner card should change based on the type of partner 
Above the Partner EntityHeader Card, we want to add a label to provide clarity and context on the partner.

- If partner type = gallery, use “At gallery”
- If partner type = Institution or Institutional Seller, use “At institution”
- If partner type = auction, or any other partner type, [Do not show section]

Screen Shot 2019-07-09 at 11 41 12 AM

Screen Shot 2019-07-09 at 1 15 50 PM

# [PURCHASE-1248] Update SellerInfo copy to adjust based on availability
To add clarity to the Partner Name below the Availability, we want to add either “Sold by [Partner Name]” or “At [Partner Name]” depending on the availability.

- On loan, Permanent collection, Not for sale, and [No Availability], will use: “At [Partner Name]”
- Any commercial availability + Sold, will use: “Sold by [Partner Name]”
- Bidding closed will not show any partner name.

Screen Shot 2019-07-09 at 1 12 43 PM

Screen Shot 2019-07-09 at 1 14 17 PM

[PURCHASE-1238]: https://artsyproduct.atlassian.net/browse/PURCHASE-1238
[PURCHASE-1248]: https://artsyproduct.atlassian.net/browse/PURCHASE-1248

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/__tests__/PartnerCard-tests.tsx [hunk 1]
```diff
@@ -41,6 +41,21 @@ describe("PartnerCard", () => {
...
+      </Theme>
+    )
+
+    expect(
+      component
+        .find(Sans)
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"At gallery"`)
+  })
```
Comment (ds300) [inline src/lib/Scenes/Artwork/Components/__tests__/PartnerCard-tests.tsx]: Should we do a test for the other partner types too?

Comment (lilyfromseattle) [reply src/lib/Scenes/Artwork/Components/__tests__/PartnerCard-tests.tsx]: Sure

**Comment by ds300** [comment]
Thanks for adding those tests!

One final question: I noticed this in the SellerInfo acceptance criteria:

> Bidding closed will not show any partner name.

Has this been taken into account?

**Comment by ds300** [comment]
Yay thanks Lily! Great work on the tests!

**Pull request merged**